soc/arm/st_stm32/stm32f1/dts_fixup.h

```diff
@@ -153,13 +153,23 @@
 #define DT_SPI_3_CLOCK_BITS		DT_ST_STM32_SPI_40003C00_CLOCK_BITS
 #define DT_SPI_3_CLOCK_BUS		DT_ST_STM32_SPI_40003C00_CLOCK_BUS
 
+#ifdef USB_OTG_FS
+#define DT_USB_BASE_ADDRESS		DT_ST_STM32_OTGFS_50000000_BASE_ADDRESS
+#define DT_USB_IRQ			DT_ST_STM32_OTGFS_50000000_IRQ_OTGFS
+#define DT_USB_IRQ_PRI			DT_ST_STM32_OTGFS_50000000_IRQ_OTGFS_PRIORITY
+#define DT_USB_NUM_BIDIR_ENDPOINTS	DT_ST_STM32_OTGFS_50000000_NUM_BIDIR_ENDPOINTS
+#define DT_USB_RAM_SIZE			DT_ST_STM32_OTGFS_50000000_RAM_SIZE
+#define DT_USB_CLOCK_BITS		DT_ST_STM32_OTGFS_50000000_CLOCK_BITS
+#define DT_USB_CLOCK_BUS		DT_ST_STM32_OTGFS_50000000_CLOCK_BUS
+#elif USB_OTG_HS
```

Comment:
It should be `#elif USB`. These macros are defined for USB 2.0 FS device interface (STM32F102/STM32F103). `USB_OTG_HS` is for SoCs with OTG high speed controller.

Reply:
The STM32F102 and F103 have the USB full-speed peripheral (i.e: Chapter 23 of the datasheet)
The STM32F105 and F107 have the USB OTG_FS peripheral (i.e: Chapter 28 of the datasheet)

I think this is correct?... apologies if not.

Reply:
> I think this is correct?

Yes. There is no USB OTG_HS peripheral in STM32F1xx series.